db/seeds.rb

```diff
@@ -1,7 +1,5 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
+User.create!(name:  "管理者",
+             email: "admin@admin.com",
```

Comment:
実在しないことが保証されているドメインを使用した方が良いでしょう。
```suggestion
             email: "admin@example.com",
```